invenio_app_ils/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -19,6 +19,20 @@
         "items": {
           "type": "keyword"
         },
+        "keywords": {
+          "type": "object",
```

Comment:
keywords should be a list in the end and in general in the document a list of $ref

Reply:
I left a comment regarding this here: https://github.com/inveniosoftware/invenio-app-ils/pull/214#discussion_r269573955

Do you think it would make any sense to have keywords as a `nested` type instead?